Add section on Client identity (#83)
tfpauly/privacy-proxy: draft-private-access-tokens.md

third-party to store these secrets, but it is a requirement of the PAT
architecture that the Mediator not be able to learn these secrets.

## Client Identity

The PAT architecture does not enforce strong constraints around the definition
of a Client identity and allows it to be defined entirely by a Mediator. If a
user can create an arbitrary number of Client identities that are accepted by

> one or more Mediators, a malicious user can easily game this architecture and

Reviewer: ```suggestion
one or more Mediators, a malicious user can easily abuse the system to
```

Reviewer 2: I like this suggestion

defeat the Issuer's ability to enforce per-Client policies.

> These multiple identities could be Sybil or true identities.

Reviewer 2: Sybil?

A Mediator alone is responsible for detecting and weeding out Sybil Client
identities in the PAT architecture. An Issuer relies on a Mediator's reputation;
as explained in {{properties}}, the correctness of the architecture hinges on
Issuers issuing tokens to only known and reputable Mediators.

Users have multiple true identities on the Internet however, and as a result, it

> seems possible for a user to game this architecture without having to create

Reviewer: ```suggestion
seems possible for a user to abuse the system without having to create
```

fake identities. For instance, a user could use multiple Mediators,
authenticating with each one using a different true identity.

The PAT architecture offers no panacea against this potential abuse.  We note
however that the usages of PATs will cause the ecosystem to evolve and offer
practical mitigations, such as:

- An Issuer can learn the properties of a Mediator - specifically, which stable
  Client identifier is authenticated by the Mediator - to determine whether the
  Mediator is acceptable for an Origin.

- An Origin can choose an Issuer based on the types of Mediators accepted by the
  Issuer, or the Origin can communicate its constraints to the designated
  Issuer.

- An Origin can direct a user to a specific Issuer based on client properties

>   that are visible. For instance,

Reviewer: ```suggestion
  that are visible. For instance, properties that are observable in the HTTP User Agent string.
```
I wasn't sure what you had in mind, but maybe this works?
